packages/stacked_generator/lib/src/generators/forms/stacked_form_content_generator.dart

```diff
@@ -1,14 +1,37 @@
+import 'package:stacked_generator/src/generators/base_generator.dart';
 import 'package:stacked_generator/src/generators/forms/form_generator_util.dart';
 import 'package:stacked_generator/src/generators/forms/form_view_config.dart';
 
 /// A generator that creates the form code based on the configs passed in
-class StackedFormContentGenerator {
+class StackedFormContentGenerator implements BaseGenerator {
   final FormViewConfig _formViewConfig;
-   late FormGeneratorUtil _util;
   StackedFormContentGenerator(this._formViewConfig);
 
+  @override
   String generate() {
-    _util = FormGeneratorUtil(formViewConfig: _formViewConfig);
-    return _util.generate();
+    final formGeneratorUtil = FormGeneratorUtil(formViewConfig: _formViewConfig)
+      ..writeHeaderComment()
+      ..generateImports()
+      ..generateValueMapKeys()
+      ..generateDropdownItemsMap()
+      ..generateTextEditingControllerItemsMap()
+      ..generateFocusNodeItemsMap()
+      ..generateValidationFunctionsFromAnnotation()
+      ..writeMixinSignature()
+      ..generateTextEditingControllersForTextFields()
```

Comment:
All `generate` commands should change to `add` to make sense in the domain language we're building in stacked.